src/App.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable*/
```

Comment:
### Your Summary
![summary](https://user-images.githubusercontent.com/93039359/187048250-e07ffeaa-82b6-44c2-84b7-3ab2f884c6ce.png)

### Descriptive Summary
![descriptive-summary](https://user-images.githubusercontent.com/93039359/187048253-65aa7051-a944-4a13-962d-bf933568f673.png)

- [ ] Your Summary is not descriptive. Please add a descriptive summary to make your pull request look professionl :blush: